src/shared-components/Card/ListItem/ListItemsSelect/ListItemSelect.tsx

```diff
@@ -0,0 +1,35 @@
+import { Checkbox } from '@mui/material';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import { GenericListItems } from '../GenericListItems';
+import { useState } from 'react';
+
+export const ListItemSelect = ({
+  title,
+  defaultSelected = true,
+}: {
+  title: string;
+  defaultSelected?: boolean;
+}) => {
+  const [isSelected, setIsSelected] = useState(defaultSelected);
```

Comment:
It will not be possible to work with this component if you keep state inside of it. Instead, you will need to keep the state in the parent component. 
ListItemSelect should accept the following props: title, isSelected, setIsSelected.
Remove state, handleSelectChange should use `setIsSelected` method to update the isSelected value (don't change `handleSelectChange`).

Reply:
Fixed